contrib/inventory/ec2.py

```diff
@@ -253,12 +254,18 @@ def read_settings(self):
 
         # Regions
         self.regions = []
-        configRegions = config.get('ec2', 'regions')
+        if config.has_option('ec2', 'regions'):
```

Comment:
That would be ideal solution but it would take much more effort. I have just implemented the missing defaults in the same style like the others. In the next step (and PR) we can do what you are proposing. If the code works for you, let's merge it now and discuss other changes in the IRC channel.